src/Store/VimStoreConnector.re

```diff
@@ -59,6 +59,7 @@ let start =
       setClipboardText,
     ) => {
   let (stream, dispatch) = Isolinear.Stream.create();
+  let hasInitialized = ref(false);
```

Comment:
```suggestion
  let libvimHasInitialized = ref(false);
```

It's not immediately clear what exactly this refers to. I first thought it might refer to `Actions.Init` having been processed, which would be really hacky, but still... You really can't be too specific with global mutable variables.